server/e2e/gql_field_test.go

```diff
@@ -627,3 +627,83 @@ func TestClearFieldDefaultValue(t *testing.T) {
 
 	assert.Equal(t, []any{nil, nil, nil}, dv)
 }
+
+func TestGuessSchemaFields(t *testing.T) {
```

Comment:
Create a helper function to call the GuessSchemaFields query 
Different test cases are missing. Please consider: 
- geojson
- Try the logic on an already existing schema